src/builtin.c

```diff
@@ -210,7 +210,7 @@ static jv f_ ## name(jq_state *jq, jv input, jv a, jv b, jv c) { \
 }
 #define LIBM_DDDD_NO(name) \
 static jv f_ ## name(jq_state *jq, jv input, jv a, jv b, jv c) { \
-  jv_free(c) \
+  jv_free(c); \
```

Comment:
This is a "minor buglet"? lol

Reply:
How did this compile? 🤔

Reply:
@wader This code is only used on systems that don't have one of the math functions with three arguments.
The only math function with 3 arguments that jq supports is [`fma`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/fma.html).
I guess all our CI targets have `fma`.

Reply:
Aha i see, wonder if there is a sane way to do a CI build that disables all conditionally built builtins just to see if compiles at least 🤔